app/assets/stylesheets/_shirts.scss

```diff
@@ -0,0 +1,21 @@
+div.shirt-banner {
+  margin-bottom: $base-spacing;
+  overflow-x: hidden;
+  white-space: nowrap;
+
+  a {
+    @include animation(marquee 20.0s linear infinite);
+    color: white;
+    display: inline-block;
+    font-style: italic;
+    font-weight: bold;
+    padding-left: 100%;
+    text-decoration: none;
+    text-transform: uppercase;
+  }
+}
+
+@include keyframes(marquee) {
+  0% { @include transform(translate(0, 0)); }
+  100% { @include transform(translate(-100%, 0)); }
+}
```

Comment:
That's a bummer, man. It would have been _soooooo on brand_.

On Sunday, February 21, 2016, Edward Loveall <notifications@github.com>
wrote:

> In app/assets/stylesheets/_shirts.scss
> <https://github.com/gabebw/hotline-webring/pull/39#discussion_r53569461>:
>
> > +  a {
> > +    @include animation(marquee 20.0s linear infinite);
> > +    color: white;
> > +    display: inline-block;
> > +    font-style: italic;
> > +    font-weight: bold;
> > +    padding-left: 100%;
> > +    text-decoration: none;
> > +    text-transform: uppercase;
> > +  }
> > +}
> > +
> > +@include keyframes(marquee) {
> > +  0% { @include transform(translate(0, 0)); }
> > +  100% { @include transform(translate(-100%, 0)); }
> > +}
>
> I guess I just didn't feel comfortable using the old element, even if it
> might work. If it works in all browsers though I'm not totally opposed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gabebw/hotline-webring/pull/39/files#r53569461>.
>


-- 


-- 
@adarshp <http://www.twitter.com/adarshp>
313.454.1515 (mobile)
See when I am free: http://bit.ly/adarshcal